openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -523,6 +525,12 @@ def run(self, *, dry=False, verbose=True,
             checkpoint_storage=sim_settings.checkpoint_storage,
         )
 
+        #  b. Write out a PDB containing the subsampled hybrid state
+        traj = mdtraj.Trajectory(
```

Comment:
I removed the new utils bits (for now at least) and opted for this since it's the best backwards compatibility we're going to get.